src/core/analytics.js

```diff
@@ -838,6 +842,12 @@ class Analytics {
       } catch (e) {
         handleError(e);
       }
+      // If cookie consent is enabled attach the denied consent group Ids to the context
+      if (fetchCookieConsentState(this.cookieConsentOptions)) {
```

Comment:
can we make use that context.consentManagement cannot be overriden by cstom apiOptions like the library details please?

Reply:
Adressed